docs/sphinx/source/developer/database.rst

```diff
@@ -0,0 +1,8 @@
+Database Structure
+-------------------
+
+The Database is a SQLite3 Database, the Tables are specified in each Packages
+model.py and in a scheme it looks like:
+
+.. image:: developer/img/diagramm_domain.svg
+  :target: _images/diagramm_domain.svg
```

Comment:
Das Bild ist bei mir kaputt

Reply:
Das liegt daran das die dateien in die master dateien includiert werden und da müssen die pfade vom master aus angegeben werden